README.md

```diff
@@ -1,15 +1,149 @@
+- **dbt Version**: This dbt package requires you have a functional dbt project that utilizes a dbt version within the respective range `>=1.0.0, <2.0.0`.
+
+## Step 2: Installing the Package
+Include the following github package version in your `packages.yml`
+> Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+```yaml
+packages:
+  - package: fivetran/app_reporting
+    version: [">=0.1.0", "<0.2.0"]
+```
+## Step 3: Configure Your Variables
+### Database and Schema Variables
+By default, this package looks for your app platform data in your target database. If this is not where your app platform data is stored, add the relevant `_database` variables to your `dbt_project.yml` file (see below).
+
+By default, this package also looks for specific schemas from each of your connectors. The schemas from each connector are highlighted in the code snippet below. If your data is stored in a different schema, add the relevant `_schema` variables to your `dbt_project.yml` file (see below).
+
+Additionally, by default, this package will also use the default source table names. If any of your tables are named differently, add the relevant `_identifier` variables to your `dbt_project.yml` file. For example, if your `app` table is named `app_usa` then you can set a variable for `app_identifier` to be `app_usa`. 
+
+```yml
+vars:
+  apple_store_schema: itunes_connect
+  apple_store_database: your_database_name
+  <default_apple_store_source_table_name>_identifier: your_table_name
+  
+  google_play_schema: google_play
+  google_play_database: your_database_name 
+  <default_google_play_source_table_name>_identifier: your_table_name
+```
+
+### Configuring Components
+Your app platform connectors might not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in your respective app platforms or have actively excluded some tables from your syncs.
+
+If you use subscriptions and have the follow tables enabled for:
+- Apple App Store
+    - `sales_subscription_event_summary`
+    - `sales_subscription_summary`
+- Google Play
+    - `<insert google tables>`
+
+Add the following variables to your dbt_project.yml file
+
+```yml
+vars:
+  apple_store__using_subscriptions: true # by default this is assumed to be false
+```
+## (Recommended) Step 4: Additional Configurations
+### Change the Build Schema
+By default this package will build all models in your <target_schema>. This behavior can be tailored to your preference by making use of custom schemas. We highly recommend use of custom schemas for this package, as multiple sources are involved. To do so, add the following configuration to your `dbt_project.yml` file:
```

Comment:
by default won't the packages write to custom schemas? like the google play source package will write to `<target_schema>_stg_google_pay` 

also we should probably add a model config below for this package (so like `app_reporting: +schema: ...`)

Reply:
oh nice catch! I'll update the doc and tag you in the change

Reply:
also I think it should be like 
`<target_schema>_apple_store` and the tables will be like `stg_google_play`?